spec/unit/locales_validation_spec.rb

```diff
@@ -0,0 +1,7 @@
+RSpec.describe "LocalesValidation", type: :model do
+  it "validates all locale files" do
+    checker = RailsTranslationManager::LocaleChecker.new("config/locales/*.yml")
+
+    expect(checker.validate_locales).to(be_truthy)
```

Comment:
the bracket looks strange, should it be `expect(checker.validate_locales).to be_truthy`

Reply:
Done!